core/domain/topic_services.py

```diff
@@ -1763,3 +1776,37 @@ def get_chapter_counts_in_topic_summaries(
         })
 
     return topic_chapter_counts_dict
+
+
+def get_published_story_exploration_ids(
+    topic_id: Optional[str] = None
+) -> List[str]:
+    """Returns a list of each exploration id linked to each published chapter.
```

Comment:
...each published chapter in a published story.

(Then you don't need the next sentence.)

Reply:
Sorry. I was trying not to create a run-on sentence.

Reply:
I ended up rewording the sentence to, "Returns a list of each exploration id linked to each published story chapter that belongs to topic(s)." Since it's impossible to publish a story's chapter without publishing the story at the same time, I don't think it's entirely necessary to separately state that the story is published. What do you think?

Reply:
Yup seems fine, thanks!

Reply:
Done.